### ISSUE TITLE
Alert role: Aria-atomic="true" missing on error container

### ISSUE THREAD

**Issue by nul800sebastiaan** [umbraco/Umbraco-CMS.Accessibility.Issues#12]
Aria-atomic="true" missing on error container

**Comment by RachBreeze** [comment]
Hi @DannyLancaster 
I've taken a look at this and it would be good to catch up. aria-live is used on the login page and on the notifications pane.

On the login page it has:
`                        <div ng-messages="vm.loginForm.$error" class="control-group" aria-live="assertive">
                            <p ng-message="auth" class="text-error" role="alert" tabindex="0">{{vm.errorMsg}}</p>
                        </div>
`

With one error in it.

On the notification page the aria alerts are set to:

`<ul ng-if="notifications" class="umb-notifications__notifications" aria-live="assertive" aria-relevant="additions">`

Which feels at odds to 'aria-atomic="true" '

Apologies if I've missed an aria-live element.

This umbraco v11 :-)
